README.md

```diff
@@ -1,11 +1,36 @@
-# Minswap tokens
+# 😽 Minswap tokens
+
+## Overview
+
 The merge of deprecated verified-tokens and market cap repositories, which contains a list of tokens, exposes APIs for transparent access to circulating supply and total supply.
 
-## Requirements
-For tokens to be verified, ensure your token has a pool with at least **1000 ADA TVL** and follow the structures stated in the instructions below. Any token that has been verified does not meet the requirements in the future would still be unverified.
+As of latest update, we consider `circulating = maxSupply - treasury - burn` and `total = maxSupply - burn` as standard formulas for calculating marketcap information.
+
+In cases where `circulatingOnChain` is provided directly according to the asset's quantity on-chain or through external APIs, the `circulating` is the value of `circulatingOnChain`.
+
+For tokens providing with `treasuryNft`, the `circulating` is the quantity of the asset in the latest address containing it.
```

Comment:
`in addresses containing it`. In default `blockFrostAdapter`, it gets amount in first address holding it.